src/main/java/spoon/reflect/path/CtRole.java

```diff
@@ -87,7 +87,9 @@
 	JAVADOC_TAG_VALUE,
 	POSITION,
 	SNIPPET,
-	ACCESSED_TYPE;
+	ACCESSED_TYPE,
+	IMPORT_REFERENCE,
+	IMPORT_ALL_TYPE_MEMBERS;
```

Comment:
this is really too low level compared to the concept of role. Is it possible to remove it?